sklearn/metrics/tests/test_classification.py

```diff
@@ -366,8 +397,6 @@ def test_matthews_corrcoef():
     y_true_inv = ["b" if i == "a" else "a" for i in y_true]
 
     assert_almost_equal(matthews_corrcoef(y_true, y_true_inv), -1)
-    y_true_inv2 = label_binarize(y_true, ["a", "b"]) * -1
```

Comment:
I think https://github.com/scikit-learn/scikit-learn/pull/8377 should be merged before this PR, which would reduce its scope. I already had @jnothman's +1 maybe @GaelVaroquaux you can have a look?